script-generator/src/main/java/com/coolbitx/wallet/signing/scriptlib/NearScript.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.coolbitx.wallet.signing.scriptlib;
+
+import com.coolbitx.wallet.signing.utils.ScriptArgumentComposer;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler;
+import com.coolbitx.wallet.signing.utils.ScriptData;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler.HashType;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler.SignType;
+import com.coolbitx.wallet.signing.utils.ScriptData.Buffer;
+import com.google.common.base.Strings;
+
+public class NearScript {
+
+    public static void listAll() {
+        System.out.println("Near: \n" + getNearScript() + "\n");
+    }
+
+    public static String getNearScript() {
+
+        ScriptArgumentComposer sac = new ScriptArgumentComposer();
+        ScriptData signerLength = sac.getArgument(1);
+        ScriptData signer = sac.getArgumentVariableLength(68);
+        ScriptData publicKey = sac.getArgument(32);
+        ScriptData nonce = sac.getArgument(8);
+        ScriptData receiverLength = sac.getArgument(1);
+        ScriptData receiver = sac.getArgumentVariableLength(68);
+        ScriptData blockHash = sac.getArgument(32);
+        ScriptData deposit = sac.getArgument(16);
+        ScriptData amount = sac.getArgument(10);
+
+        String script = new ScriptAssembler()
+            .setCoinType(0x018d)
+            .copyArgument(signerLength)
+            .copyString("000000")
+            .setBufferInt(signerLength, 2, 64)
+            .copyArgument(signer)
+            .copyString("00")
+            .copyArgument(publicKey)
+            .baseConvert(nonce, Buffer.TRANSACTION, 8, ScriptAssembler.binaryCharset, ScriptAssembler.inLittleEndian)
+            .copyArgument(receiverLength)
+            .copyString("000000")
+            .setBufferInt(receiverLength, 2, 64)
+            .copyArgument(receiver)
+            .copyArgument(blockHash)
+            .copyString("0100000003")
+            .baseConvert(deposit, Buffer.TRANSACTION, 16, ScriptAssembler.binaryCharset, ScriptAssembler.inLittleEndian)
+            .showMessage("NEAR")
+            .showAddress(receiver)
+            .showAmount(amount, 10)
```

Comment:
please use deposit argument in the showAmount method to secure the verification between signing data and screen display.